### PR TITLE
Add country-select-js w/ git auto-update via single package.json

### PR DESCRIPTION
**Pull request by ralphchung**
Pull request for issue: #11462

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/mrmarkfrench/country-select-js
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/country-select-js
 * License and its reference: [MIT](https://github.com/mrmarkfrench/country-select-js/blob/master/package.json#L16)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 12
   - Count of stars: 143
   - Count of forks: 65
 * NPM download stats (optional):
   - Downloads in the last day: 67
   - Downloads in the last week: 220
   - Downloads in the last month: 1312

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by ralphchung** [comment]
@sashberd I think you meant I should edit this PR and I have done that. Please check it. Thank you!

File: ajax/libs/country-select-js/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "country-select-js",
+  "filename": "build/css/countrySelect.min.css",
```
Comment (extend1994) [inline ajax/libs/country-select-js/package.json]: I think the main files should be `countrySelect.min.js`. Refer https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/intl-tel-input/package.json#L3. It's the base library, which is hosted in cdnjs , of this library.

Besides this, filename the path to file should be relative path to `basePath` in auto-update config, so here would be `js/countrySelect.min.js`.

**Comment by extend1994** [comment]
@ralphchung When you open the pull request "via single package.json", there is no need to specify the version.

File: ajax/libs/country-select-js/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "keywords": [
+    "country",
+    "international",
+    "jQuery"
+  ],
+  "author": "Mark French",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/mrmarkfrench/country-select-js.git",
```
Comment (sufuf3) [inline ajax/libs/country-select-js/package.json]: Next time you can use `git` protocol.
Let's bot get new versions via git protocol.
Thanks

Comment (PeterDaveHello) [reply ajax/libs/country-select-js/package.json]: No mind, it's fine to me.

**Pull request merged**